dev/free_disk_space_container

```diff
@@ -21,13 +21,24 @@ echo "=================================="
 echo "Free up disk space on CI system"
 echo "=================================="
 
-echo "Listing 100 largest packages"
-dpkg-query -Wf '${Installed-Size}\t${Package}\n' | sort -n | tail -n 100
+echo "Listing top 100 largest packages (from large to small)"
+printf "Installed-Size\tPackage\n"
+dpkg-query -Wf '${Installed-Size}\t${Package}\n' | sort -n -r | head -n 100
 df -h
 
 echo "Removing large packages"
 rm -rf /__t/CodeQL
 rm -rf /__t/go
 rm -rf /__t/node
 
+apt-get remove --purge -y '^aspnet.*'
```

Comment:
It seems to be out of scope of the PR title, @panbingkun . Could you revise the PR title?

Reply:
Done, but the title is `too long` and it's not easy to summarize it into one simple sentence, 😄

Reply:
Thank you for updating.